src/neo4j_genai/embeddings/sentence_transformers.py

```diff
@@ -0,0 +1,17 @@
+from neo4j_genai.embedder import Embedder
+
+
+class SentenceTransformerEmbeddings(Embedder):
+    def __init__(self, model="all-MiniLM-L6-v2"):
+        try:
+            from sentence_transformers import SentenceTransformer
+        except ImportError:
+            raise ImportError(
+                "Could not import sentence_transformers python package. "
+                "Please install it with `pip install sentence-transformers`."
+            )
+
+        self.model = SentenceTransformer(model)
```

Comment:
Will the user be interested in other arguments to SentenceTransformer ([see reference](https://sbert.net/examples/applications/computing-embeddings/README.html?highlight=sentencetransformer#sentence_transformers.SentenceTransformer)). If so, maybe we should use **args, **kwargs?

Reply:
e63b7fdf4745245cb0954474364d980e3badc7bd